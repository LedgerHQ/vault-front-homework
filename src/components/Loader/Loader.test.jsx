import { Loader } from "./Loader";
import { render } from "@testing-library/react";

describe("Loader", () => {
  it("should match snapshot", () => {
    const { container } = render(<Loader />);
    expect(container).toMatchSnapshot();
  });
});
