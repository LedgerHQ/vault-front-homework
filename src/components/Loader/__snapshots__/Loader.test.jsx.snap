// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader should match snapshot 1`] = `
<div>
  <div
    class="flex items-center gap-1"
  >
    <svg
      class="animate-spin"
      height="24"
      stroke="black"
      viewBox="0 0 38 38"
      width="24"
      xmlns="http://www.w3.org/2000/svg"
    >
      <g
        fill="none"
        fill-rule="evenodd"
      >
        <g
          stroke-width="2"
          transform="translate(1 1)"
        >
          <circle
            cx="18"
            cy="18"
            r="18"
            stroke-opacity=".5"
          />
          <path
            d="M36 18c0-9.94-8.06-18-18-18"
          />
        </g>
      </g>
    </svg>
  </div>
</div>
`;
